Mathlib/MeasureTheory/Measure/Typeclasses.lean

```diff
@@ -552,12 +552,22 @@ lemma sfinite_sum_of_countable {ι : Type*} [Countable ι]
   · rw [Function.extend_apply' _ _ _ hn, Pi.zero_apply]
     infer_instance
 
-instance {ι : Type*} [Countable ι] (m : ι → Measure α) [∀ n, SFinite (m n)] :
-    SFinite (Measure.sum m) := by
+instance [Countable ι] (m : ι → Measure α) [∀ n, SFinite (m n)] : SFinite (Measure.sum m) := by
   change SFinite (Measure.sum (fun i ↦ m i))
   simp_rw [← sum_sFiniteSeq (m _), Measure.sum_sum]
   apply sfinite_sum_of_countable
 
+instance instSFiniteAdd [SFinite μ] [SFinite ν] : SFinite (μ + ν) := by
```

Comment:
Is there a reason why you're naming the instance? Typically, we use unnamed instances unless there is a good reason.

Same thing below.

Reply:
The idea was to avoid the gigantic auto-generated names that pollute the doc search. But if I recall correctly there was an update and this is no longer a concern (or not as much) and I should go back to not naming instances.